.github/workflows/empty-worksapce-smoke-test-on-minikube-ubi8.yaml

```
@@ -0,0 +1,110 @@
+#
+# Copyright (c) 2019-2021 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+#
+
+name: Empty workspace smoke test on udi8
+on:
+  pull_request:
+    paths-ignore:
+      - '**/*.md'
+      - .devfile.yaml
+      - LICENSE
+      - '.rebase/*'
+
+env:
+   USERSTORY: CloneGitRepoAPI
+   TS_API_TEST_KUBERNETES_COMMAND_LINE_TOOL: kubectl
+   DEPLOYMENT_TIMEOUT: 90s
+
+jobs:
+  workspace-api-tests-on-minikube:
+    runs-on: ubuntu-22.04
+    steps:
+
+    - name: Checkout
+      uses: actions/checkout@master
+
+    - name: Login to Quay
+      uses: docker/login-action@v1
+      with:
+        registry: quay.io
+        username: ${{ secrets.QUAY_USERNAME }}
+        password: ${{ secrets.QUAY_PASSWORD }}
+
+    - name: Get PR number
+      id: get_pr_number
+      run: |
+        pr_number=$(echo $GITHUB_REF | awk 'BEGIN { FS = "/" } ; { print $3 }')
+        echo "PR_NUMBER=$pr_number" >> $GITHUB_ENV
+
+    - name: Build base image
+      run: |
+        cd base/ubi8 && docker buildx build --platform linux/amd64 -t quay.io/devfile/base-developer-image:${{ env.PR_NUMBER }} .
+
+    - name: Build universal image
+      run: |
+        cd universal/ubi8 && docker buildx build --platform linux/amd64 -t quay.io/devfile/universal-developer-image:${{ env.PR_NUMBER }} .
+
+    - name: Push universal base image
+      run: |
+        docker push quay.io/devfile/universal-developer-image:${{ env.PR_NUMBER }}
+
+    - name: Start minikube cluster
+      id: run-minikube
+      uses: che-incubator/setup-minikube-action@next
+      with:
+        minikube-version: v1.23.2
```

v1.23.2 is almost 2 years old. Is there any concern using the latest ([v1.30.1](https://github.com/kubernetes/minikube/releases/tag/v1.30.1))?

```suggestion
        minikube-version: v1.30.1
```